site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,483 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I'm pleased to announce the release of intake-esm version 2020.3.16. This is a new release with bug fixes and new features. Everyone is invited to give it a try, and make their thoughts, suggestions, feedback known! This blogpost outlines these changes. Full changelog is available [here](https://intake-esm.readthedocs.io/en/latest/changelog.html#intake-esm-v2020-03-16).\n",
+    "\n",
+    "\n",
+    "On GitHub: https://github.com/NCAR/intake-esm\n",
+    "\n",
+    "Documentation: https://intake-esm.readthedocs.io/\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "Intake-esm can be installed from PyPI with pip:\n",
+    "\n",
+    "```bash\n",
+    "python -m pip intall intake-esm --upgrade\n",
+    "```\n",
+    "\n",
+    "It is also available from conda-forge channel for conda isntallations:\n",
+    "\n",
+    "```bash\n",
+    "conda install -c conda-forge intake-esm\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## New Features\n",
+    "\n",
+    "\n",
+    "### Enhanced search: enforce query criteria via `require_all_on` argument\n",
+    "\n",
+    "By default intake-esm's `search()` method returns entries that fulfill **any of the criteria** specified in the query. Today intake-esm can return entries that fulfill **all query criteria** when the user supplies the `require_all_on` argument. `require_all_on` consits of attributes/dataframe columns to use when enforcing the query criteria.\n",
+    "\n",
+    "`require_all_on argument` is best explained with examples:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "pangeo-cmip6-ESM Collection with 235624 entries:\n",
+       "\t> 15 activity_id(s)\n",
+       "\n",
+       "\t> 32 institution_id(s)\n",
+       "\n",
+       "\t> 69 source_id(s)\n",
+       "\n",
+       "\t> 101 experiment_id(s)\n",
+       "\n",
+       "\t> 135 member_id(s)\n",
+       "\n",
+       "\t> 29 table_id(s)\n",
+       "\n",
+       "\t> 313 variable_id(s)\n",
+       "\n",
+       "\t> 10 grid_label(s)\n",
+       "\n",
+       "\t> 235624 zstore(s)\n",
+       "\n",
+       "\t> 60 dcpp_init_year(s)"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Open collection for CMIP6 data hosted on Google Storage\n",
+    "import intake\n",
+    "\n",
+    "url = \"https://git.io/JvP9r\"\n",
+    "col = intake.open_esm_datastore(url)\n",
+    "col"
+   ]
+  },
+  {
```

Comment:
Add some explanatory text for what the next code cell block is telling you (i.e., better to tell the reader what they *are about* to see, rather than what they *just saw*).  Something like:

> Let's define a query for our collection that requests multiple `variable_id`s and multiple `experiment_id`s from the `Omon` `table_id`, all from 3 different `source_id`s: